src/js/hooks/productSupplier/form/useProductSupplierForm.js

```diff
@@ -60,6 +63,14 @@ const useProductSupplierForm = () => {
           label: productSupplier?.ratingTypeCode,
         }
         : undefined,
+      preferenceType: defaultPreferenceType?.preferenceType ? {
+        id: defaultPreferenceType.preferenceType.id,
+        label: defaultPreferenceType.preferenceType.name,
+        value: defaultPreferenceType.preferenceType.id,
+      } : undefined,
+      validityStartDate: defaultPreferenceType?.validityStartDate ?? undefined,
+      validityEndDate: defaultPreferenceType?.validityEndDate ?? undefined,
+      bidName: defaultPreferenceType?.bidName ?? undefined,
```

Comment:
it would be better to store all those values under the same key - it will allow us to clear/reset those values simultaneously.

I mean adding:
(here, in the default values)
```
      defaultPreferenceType: {
        ...defaultPreferenceType,
        preferenceType: {
          id: defaultPreferenceType?.preferenceType?.id,
          label: defaultPreferenceType?.preferenceType?.name,
          value: defaultPreferenceType?.preferenceType?.id,
        },
      },
```
and then in `DefaultPreferenceType.jsx` for each controller prepend: `defaultPreferenceType`